x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/update_rules_route.ts

```diff
@@ -118,6 +118,7 @@ export const updateRulesRoute = (router: IRouter) => {
           note,
           version,
           lists,
+          actions: throttle === 'rule' ? actions : [], // Only enable actions if throttle is empty, otherwise we are a notification and should not enable it
```

Comment:
@FrankHassanabad I think this logic might be reversed? It doesn't seem to match the comment. Consequence is that if I disable/enable a rule, `throttle` comes through as `null` and so our actions are updated to `[]`. Behavior is that one cannot update a rule without breaking its actions.

Reply:
The comment is incorrect for sure. I can update that.

For the enable/disable a rule, that should come through as a PATCH, let me check the logic on that and sure I am not losing actions as that would be a very bad bug there.

Reply:
Makes sense. I tested PUT and it does not overwrite actions, it's just the PATCH route.

Reply:
Per zoom conversations we removed this to avoid issues and because 100% of the time the UI utilizes the full update and it makes more sense were PATCH calls are just for small items such as enable/disable or name, title, etc... not really two things at once such as actions and throttle which are more involved and requires both to be present.